packages/2019-transportation/src/components/DisturbanceStops/disturbanceStopsLineChartMeta.js

```diff
@@ -1,20 +1,19 @@
 import React from "react";
 import { Collapsable } from "@hackoregon/component-library";
 
-import TransportationCardVisualization from "./TransportationCardVisualization";
+import DisturbanceStopsLineChartVisualization from "./DisturbanceStopsLineChartVisualization";
 
-const transportationCardMeta = (/* data */) => ({
-  title: "(Story Card #1, Issue #000) Transportation Card Title",
-  slug: "transportation-card",
+const DisturbanceStopsMeta = (/* data */) => ({
+  title: "Total Time of Disturbance Stops",
+  slug: "disturbance-stops",
```

Comment:
This should have a different slug than the other card. You can see if you click "Share" and go that link, that it will direct you to the non-line-chart card

Reply:
for this, i brought both visualizations together into the same card. now there's only one card and share link between the comparison map and the line chart